### PR TITLE
[docs] update list of winning solutions in data science competitions

### PR DESCRIPTION
**Pull request by Juniper-23**
Added winning solutions for LightGBM from completed Kaggle Competitions.

### CODE REVIEW

**Comment by jameslamb** [comment]
Thanks @Juniper-23 !  I'm making two changes to this pull request:

1.  changed the name of the PR to be something that describes what was done (since it will end up in a changelog: https://github.com/microsoft/LightGBM/releases/tag/untagged-1c866fba1430bac197c4
2. explicitly mentioning that this fixes #3357 so a link will be made between this PR and that issue

File: examples/README.md [hunk 1]
```diff
@@ -13,6 +13,14 @@ Machine Learning Challenge Winning Solutions
 | Place         | Competition   | Solution  | Date |
 | ------------- |:------------- | --------- | -----|
 |1st      | [M5 Forecasting - Accuracy](https://www.kaggle.com/c/m5-forecasting-accuracy) | [link](https://www.kaggle.com/c/m5-forecasting-accuracy/discussion/163684) | 2020.6 |
+|3rd      | [M5 Forecasting - Uncertainty](https://www.kaggle.com/c/m5-forecasting-uncertainty) | [link](https://www.kaggle.com/c/m5-forecasting-uncertainty/discussion/166875) | 2020.7 |
+|3rd      | [ALASKA2 Image Steganalysis](https://www.kaggle.com/c/alaska2-image-steganalysis) | [link](https://www.kaggle.com/c/alaska2-image-steganalysis/discussion/168870) | 2020.7 |
```
Comment (jameslamb) [inline examples/README.md:17]: can you please move the `2020.7` entries  above  the existing M5 forecasting entry? The list should be in chronological order, with the newest results on the top

image

File: examples/README.md [hunk 2]
```diff
@@ -13,6 +13,14 @@ Machine Learning Challenge Winning Solutions
...
+|1st      | [M5 Forecasting - Accuracy](https://www.kaggle.com/c/m5-forecasting-accuracy) | [link](https://www.kaggle.com/c/m5-forecasting-accuracy/discussion/163684) | 2020.6 |
```
Comment (jameslamb) [inline examples/README.md:18]: can you please remove this line? This entry already exists in this table

**Review by jameslamb — CHANGES_REQUESTED** [review]
Thanks for contributing to `LightGBM`! I've requested a few small changes.

I updated this branch to `master`, so you should `git pull origin changes` before making the changes.

**Comment by Juniper-23** [comment]
@jameslamb Thank You, It was my first PR so tried to put together whatever I could figure out on Google regarding contributions. :) I shall do the necessary changes and update ASAP.

**Comment by jameslamb** [comment]
no problem! congrats on your first PR and thanks for choosing LightGBM for it.

If you're interested, I have some slides and a recording of a talk I do on how to get started in open source: https://github.com/jameslamb/talks/tree/main/you-can-do-open-source

File: examples/README.md (hunk 2 above)
Comment (Juniper-23) [reply examples/README.md:18]: I have made the required changes to the first line of winning solutions. I believe there will be more winning KDD/notebooks. I shall update them sooner.

**Comment by Juniper-23** [comment]
@jameslamb I did go through the slides, found them handy and helpful. Thanks again :) I am yet to watch the video but it would be a really helpful add-on, if some common possible errors that one can comes across with (and their suggestive resolution) while using the git to push/pull code are listed out. I did see that git comes up with nice auto-suggestions that what could have possibly gone wrong but a little bit of google does help. :)

**Review by jameslamb — APPROVED** [review]
this looks great, thanks very much! I'm going to merge this, but we'd welcome other contributions to keep updating this list  (https://github.com/microsoft/LightGBM/pull/3358/files#r484571297)

**Pull request merged**